server.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+
+const app = express();
+const PORT = 3000;
```

Comment:
бывает очень полезно задавать порт в переменных окружения (когда будешь докер настраивать), поэтому тут лучше написать так
```suggestion
const PORT = process.env.PORT || 3000;
```